Tools/peg_generator/pegen/__main__.py

```diff
@@ -107,7 +107,13 @@ def generate_python_code(
     help="Suppress code emission for rule actions",
 )
 
-python_parser = subparsers.add_parser("python", help="Generate Python code")
+python_parser = subparsers.add_parser(
+    "python",
+    help=(
+        "Generate Python code, needs grammar definition with Python actions, "
+        "see `we-like-parsers/pegen`, start with `data/python.gram`"
```

Comment:
I don't think we should mention that repo. The input for the parser can be any grammar other than the general Python one. We use to generate the meta parser for example. I think linking to another repo it's actually more confusing.

This is also an internal tool without external user guarantees.

I would be on improving the help to mention a bit of this but I still prefer not to link against the original repo

Reply:
Ok, how about this?